lib/install/elm.rb

```diff
@@ -19,4 +19,9 @@
 run "./bin/yarn add --dev elm-hot-loader elm-webpack-loader"
 run "yarn run elm package install -- --yes"
 
+puts "Updating elm source location"
+source_path = File.join Webpacker::Configuration.source,
```

Comment:
[Fixed](https://github.com/bkuhlmann/webpacker/blob/elm-webpack-loader/lib/install/elm.rb#L23). Yeah, my personal limit is a 100 line column.